### PR TITLE
Extend response header autocompletion support

### PR DESCRIPTION
**Pull request by mhidalgo-bdai**
Follow-up to #81. This patch extends the logic for response header autocompletion to gRPC services that are not unary in both request and response.

### CODE REVIEW

File: spot_wrapper/testing/helpers.py [hunk 1]
```diff
@@ -3,20 +3,18 @@
 import functools
 import typing
 
-import grpc
-from bosdyn.api.header_pb2 import CommonError
 from bosdyn.api.lease_pb2 import ResourceTree
 
 
-class GeneralizedDecorator:
+class ThinDecorator:
```
Comment (amessing-bdai) [inline spot_wrapper/testing/helpers.py:9]: Any chance we can get a more descriptive name?

Comment (mhidalgo-bdai) [reply spot_wrapper/testing/helpers.py:9]: I'm not always great with names. `TransparentDecorator`? `ForwardingDecorator` is almost tautological.

Comment (amessing-bdai) [reply spot_wrapper/testing/helpers.py:9]: Naming might be the hardest thinking in programming, but I can't tell you what a `ThinDecorator` does just by reading the name. Maybe just add a usage comment?

Comment (mhidalgo-bdai) [reply spot_wrapper/testing/helpers.py]: How about `ForwardingWrapper`? https://github.com/bdaiinstitute/spot_wrapper/pull/120/commits/6901afa781bdb47f1f739a5b0d867bf9a7cd6651

**Review by amessing-bdai — APPROVED** [review]

**Pull request merged**